app/javascript/components/Tabs/Tab.tsx

```diff
@@ -0,0 +1,41 @@
+import { Slot } from "@radix-ui/react-slot";
+import * as React from "react";
+
+import { classNames } from "$app/utils/classNames";
+
+type TabProps = {
+  children: React.ReactNode;
+  isSelected?: boolean;
+  onClick?: () => void;
+  ariaControls?: string;
+  className?: string;
+  asChild?: boolean;
+};
+
+const Tab = ({ children, isSelected, onClick = () => {}, ariaControls, className, asChild = false }: TabProps) => {
+  const Component = asChild ? Slot : "div";
+
+  return (
+    <Component
+      role="tab"
+      className={classNames(
+        "opacity-70",
+        "dark:font-bold dark:text-white/70",
+        "rounded border border-border",
+        "hover:-translate-x-1 hover:-translate-y-1 hover:shadow-[0.25rem_0.25rem_0_currentColor]",
+        "transition-[opacity,transform] duration-200 ease-out",
+        {
+          "bg-white font-bold opacity-100 dark:bg-transparent dark:text-white": isSelected,
+        },
+        className,
+      )}
```

Comment:
IMO we should keep this using the legacy CSS for now and change it in the Tailwind migration - these look like button styles so we might be able to share some when we migrate button, for example

Reply:
I was able to leverage our existing tabs component by adding a variant here: https://github.com/antiwork/gumroad/pull/1862/commits/695186566de0b2705133fc1fd393aef431b4c46f